zirgen/bootstrap/src/main.rs

```diff
@@ -159,8 +159,8 @@ fn get_bazel_bin() -> PathBuf {
 }
 
 fn strip_locations_and_whitespace(text: &str) -> Result<String> {
-    // replace everything that might be a line number with '*'
-    let re = Regex::new(":[0-9]+")?;
+    // replace everything that might be a line number or a copyright date with '*'
+    let re = Regex::new("(:[0-9]+)|( [0-9]+ RISC)")?;
```

Comment:
It looks to me as though this would match text anywhere in the file, not just on the first line, and that makes me a little nervous (though it's hard to imagine meaningful problems which would occur from false positive matches). Would it be too much hassle to limit the regex to the expected copyright line?

Reply:
Yeah, it's just for checking to make sure we didn't forget to regenerate files so it doesn't have to be perfect.  I changed the regex to be a bit more specific about matching the copyright line.